libs/SmartSync/src/com/salesforce/androidsdk/smartsync/util/SyncDownTarget.java

```diff
@@ -149,7 +149,7 @@ public QueryType getQueryType() {
     public long getLatestModificationTimeStamp(JSONArray records) throws JSONException {
         long maxTimeStamp = -1;
         for (int i = 0; i < records.length(); i++) {
-            String timeStampStr = JSONObjectHelper.optString(records.getJSONObject(i), getModificationDateFieldName());
+            String timeStampStr = JSONObjectHelper.optString(records.getJSONObject(i), modificationDateFieldName);
```

Comment:
I suppose we could. This field is likely to be set only once in the lifecycle of the target (while creation), so I figured the protected member would probably be set/overridden in the constructor for the subclass and that'd be that. The getter would be used by other classes that use the target as public API to get the field name. Would you prefer making it private again? I'm okay either way.